app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -55,15 +55,15 @@ class ScrollingActivity : AppCompatActivity() {
     public override fun onStart() {
         super.onStart()
         val mAuth = (applicationContext.applicationContext as Cocktails).mAuth
-        mAuth.signInAnonymously()
-            .addOnCompleteListener { task ->
-                if (task.isSuccessful) {
-                    val user: FirebaseUser? = mAuth.currentUser
-                    // todo: einav - get user custom cocktails objects? (remember to save the object once being created). maybe use SP instead?
-                } else {
-                    Toast.makeText(applicationContext, "Error loading data", Toast.LENGTH_SHORT).show()
-                }
-            }
+//        mAuth.signInAnonymously()
```

Comment:
@enavhasson your call if you want to use mAuth (firebase authentication), if not - you should delete all mAuth  references (here and via gardel and 'Cocktails' classes) :)

Reply:
perfect